src/analysis/souffle/dataflow_graph.dl

```diff
@@ -73,4 +89,21 @@ path(from, to) :- resolvedEdge(_, from, intermediate), path(intermediate, to).
 isAccessPath(x) :- resolvedEdge(_, x, _).
 isAccessPath(y) :- resolvedEdge(_, _, y).
 
+// The 0th predecessor shall be the predecessor with the smallest src ordinal
+// number.
+orderedPredecessors(ap, for, src, 0) :-
```

Comment:
`for` is a bit confusing. Can we `owner` or some other name?

Reply:
Yep, will do.

Reply:
Done.